graphql/schema/types/task.graphql

```diff
@@ -96,6 +96,10 @@ type Task {
   logs: TaskLogLinks!
   minQueuePosition: Int!
   order: Int!
+  """
+  The originalStatus field represents the original status of the task without a display status applied.
+  """
+  originalStatus: String!
```

Comment:
If you're willing to do this in a backwards-compatible way I feel like it'd be preferable to have this align with the rest of the codebase, i.e. using `status` and `displayStatus`. Not a blocker though as I imagine that could be a wide-reaching change on the front end.

Reply:
I'm starting this now with this pr. A follow up UI pr will swap fields to use this new field. And in a future pr `status` will go back to its originall value.